WooCommerce/src/androidTest/kotlin/com/woocommerce/android/screenshots/orders/SingleOrderScreen.kt

```diff
@@ -18,20 +25,24 @@ class SingleOrderScreen : Screen {
         return OrderListScreen()
     }
 
-    fun assertSingleOrderScreen(): SingleOrderScreen {
+    private fun assertIdAndTextDisplayed(id: Int, text: String?) {
+        Espresso.onView(
+            Matchers.allOf(
+                ViewMatchers.withId(id), ViewMatchers.withText(text)
+            )
+        ).check(ViewAssertions.matches(ViewMatchers.isDisplayed()))
+    }
+
+    fun assertSingleOrderScreenWithProduct(order: OrderData): SingleOrderScreen {
         Espresso.onView(withId(R.id.toolbar))
-            .check(ViewAssertions.matches(hasDescendant(withSubstring("#"))))
-            .check(ViewAssertions.matches(isDisplayed()))
-        Espresso.onView(withId(ORDER_NUMBER_LABEL))
-            .check(ViewAssertions.matches(isDisplayed()))
-        Espresso.onView(withId(R.id.paymentInfo_total))
+            .check(ViewAssertions.matches(hasDescendant(withSubstring("#" + order.id))))
```

Comment:
WDYT of making this check more specific in the following way?

```suggestion
            .check(ViewAssertions.matches(hasDescendant(withText("Order #" + order.id))))
```

Reply:
Yup that's looks better actually, updated in https://github.com/woocommerce/woocommerce-android/pull/7302/commits/841bbae63d88cb05ddc3ee8d00c5cb31d7e63b7b